### ISSUE TITLE
Change daily quests or the rewards

### ISSUE THREAD

**Issue by johnclark96** [Starlight-Pixelmon/server-development#3]

**Comment by johnclark96** [comment]
There are some issues currently with making/working with the Pixelmon quest system and is not working properly. I heard they are suppose to be working on that soon to fix the issues it's having. Will need to wait on that or make this a VERY low priority

**Comment by Nesroht** [comment]
So just disable it for now? As it looks to be working and has configs that can be changed.

**Comment by johnclark96** [comment]
not a priority atm, we can look into it a bit later after we clean up the rest of these tasks
